Code/Tools/AssetBundler/source/utils/AssetBundlerBuildTarget.cpp

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
These had to be separate files for each? I figured having that cmake defined macro would allow you to share this file across different targets

Reply:
I tried sharing the AssetBundlerTarget.cpp but I think CMake is overwriting the `O3DE_CMAKE_TARGET` with whatever it is set to last so it just used `AssetBundler` for both.